jmxtrans/pom.xml

```diff
@@ -354,8 +350,7 @@
 									fi</script>
 							</preremoveScriptlet>
 							<requires>
-								<require>java</require>
-								<require>/usr/bin/lsb_release</require>
+								<require>java >= 1.7</require>
```

Comment:
Damn, you might well be right. Maybe it is time to drop 1.6 support. Oracle stopped public support in 2013, so it might be reasonable to upgrade to 1.7 (and try-with-resources at last!)